pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -179,42 +173,12 @@ async def run(self):
         running_in_jenkins = os.environ.get('BUILD_ID', False)
         if running_in_jenkins:
             title_update = f" {self.ocp_version} - {go_version} - el{list(el_nvr_map.keys())}"
-            if self.create_ticket:
-                title_update += ' [create-ticket]'
             if self.dry_run:
                 title_update += ' [dry-run]'
             jenkins.init_jenkins()
             jenkins.update_title(title_update)
 
-        # Do a sanity check for rhel8 module build
-        # a module build should never be tagged directly in override tag
-        # if it is, we should make sure it is available via module inheritance
-        # and then untag it from override tag
-        if 8 in el_nvr_map and 'module' in el_nvr_map[8]:
-            tag = f'rhaos-{self.ocp_version}-rhel-8-override'
-            package = parse_nvr(el_nvr_map[8])['name']
-            nvr = get_latest_nvr_in_tag(tag, package, self.koji_session)
-            if nvr:
-                raise ValueError(f'{nvr} is tagged in {tag}, please make sure it is available via module inheritance '
-                                 f'(brew list-tagged {tag} {package} --inherit). If not, use --create-tagging-ticket. '
-                                 f'Once it is available, untag it from the override tag (brew untag-build {tag} {nvr})')
-
-        # if requested, create ticket for tagging request
-        if self.create_ticket:
-            _LOGGER.info('Making sure that builds are not already tagged')
-            for el_v, nvr in el_nvr_map.items():
-                if is_latest_build(self.ocp_version, el_v, nvr, self.koji_session):
-                    raise ValueError(f'{nvr} is already the latest build, run '
-                                     'only with nvrs that are not latest or run without --create-tagging-ticket')
-
-            _LOGGER.info('Creating Jira ticket to tag golang builds in buildroots')
-            self.create_jira_ticket_for_el8(el_nvr_map, go_version)
-            return
-
-        cannot_proceed = False
-        for el_v, nvr in el_nvr_map.items():
-            if not await is_latest_and_available(self.ocp_version, el_v, nvr, self.koji_session):
-                cannot_proceed = True
+        cannot_proceed = any(await asyncio.gather(*[self.process_build(el_v, nvr) for el_v, nvr in el_nvr_map.items()]))
         if cannot_proceed:
             raise ValueError('Cannot proceed until all builds are tagged and available')
```

Comment:
```suggestion
            raise ValueError('Cannot proceed until all builds are tagged and available. Run command with --tag-builds to tag and make the builds available.')
```

Reply:
if the error comes from here, it's likely due to builds already tagged but not showing in wait-repo